docs/maintenance.md

```diff
@@ -19,31 +19,33 @@ First of all, we should have a look at the release notes in the order below.
     - Choose the next version and check the [release note](https://kubernetes.io/docs/setup/release/notes/). e.g. 1.17, 1.18, 1.19 -> 1.18, 1.19, 1.20
     - Read the [release note](https://github.com/kubernetes-sigs/controller-runtime/releases), and check which version is compatible with the Kubernetes versions.
     - Read the [kubebuilder go.mod](https://github.com/kubernetes-sigs/kubebuilder/blob/master/go.mod), and check the controller-tools version corresponding to controller-runtime.
-3. CSI spec
+1. CSI spec
     - Read the [release note](https://github.com/container-storage-interface/spec/releases) and check all the changes from the current version to the latest.
     - Basically, CSI spec should NOT be upgraded aggressively in this task.
     Upgrade the CSI version only if new features we should cover are introduced in newer versions, or the Kubernetes versions TopoLVM is going to support does not support the current CSI version.
     - For updating CSI spec, please remove `csi.proto` file then run `make generate`.
-4. CSI sidecars
+1. CSI sidecars
     - TopoLVM does not use all the sidecars listed [here](https://kubernetes-csi.github.io/docs/sidecar-containers.html).
       Have a look at `csi-sidecars.mk` first and understand what sidecars are actually being used.
     - Check the release pages of the sidecars under [kubernetes-csi](https://github.com/kubernetes-csi) one by one and choose the latest version for each sidecar which satisfies both "Minimal Kubernetes version" and "Supported CSI spec versions".
       DO NOT follow the "Status and Releases" tables in this [page](https://kubernetes-csi.github.io/docs/sidecar-containers.html) and the README.md files in the sidecar repositories because they are sometimes not updated properly.
     - Read the change logs which are linked from the release pages.
-    - Confirm diffs of RBAC between published in upstream and `deploy/manifests/base/controller.yaml`. And update it if required.
-5. Depending tools
+    - Confirm diffs of RBAC between published in upstream and follows. And update it if required.
+      - charts/topolvm/templates/controller/clusterroles.yaml
+      - charts/topolvm/templates/controller/roles.yaml
```

Comment:
Should we add `clusterrolebindings.yaml` and `rolebinding.yaml` as well?

Reply:
I prefer to use backtick to describe files.

Reply:
About `clusterrolebindings.yaml` and `rolebindg.yaml`, I guess it may update only when adding some sidecars. So we will not have to update it regularly.